mantidproject/mantid
# D2B normalisation fix

**Description of work.**

The normalisation in `SumOverlappingTubes` got broken (see the issue).
This was not captured since the tests were using data with sufficiently high counts.
The bug was revealed in a recent short measurement, where many pixels counted 0.

This PR fixes the bug, and adds more rigor to the tests.

**Report to:** G. Cuello.
 

**To test:**

- [ ] ```red = PowderDiffILLDetScanReduction(Run='540162')```
 Plot and confirm by eye, that the ratio of the highest peak and the background is about 10. The test run should be in system test data under ILL/D2B.

- [ ] ```./systemtest -R ILLPowderDiffDetScanReduction``` 



Fixes #23521 . 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
